components/tooltip/README.md

```diff
@@ -112,20 +119,35 @@ This can be done with the `boundary` attribute that allows any of the tooltip's
 
 In the following example to constrain the tooltip to the dashed boundary we can set the top boundary to `50`, the bottom boundary to `10`, the left boundary to `100`, and the right boundary to `0`.
 
+<!-- docs: start hidden content -->
 ![screenshot of a tooltip with custom boundaries](./screenshots/tooltip-boundary.png)
+<!-- docs: end hidden content -->
 
-<!-- docs: demo code autoSize:false size:small -->
+<!-- docs: demo code -->
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/button/button.js';
   import '@brightspace-ui/core/components/tooltip/tooltip.js';
 </script>
-
-<div class="offset-parent">
-	<d2l-button id="tooltip-boundary">Tooltip boundary</d2l-button>
-	<d2l-tooltip for="tooltip-boundary"
-		boundary="{&quot;top&quot;:50, &quot;bottom&quot;:10, &quot;left&quot;:100, &quot;right&quot;:0}">
-		This tooltip will not expand beyond its boundaries unless it is impossible to fit it inside
-	</d2l-tooltip>
+<style>
+  .boundary {
+    border: 1px dashed #cdd5dc;
+    border-radius: 6px;
+    box-sizing: border-box;
+    display: inline-block;
+    height: 175px;
+    padding-left: 150px;
+    padding-top: 60px;
+    position: relative;
+    width: 350px;
+  }
+</style>
+
+<div class="offset-parent boundary">
+  <d2l-button id="tooltip-boundary">Tooltip boundary</d2l-button>
+  <d2l-tooltip for="tooltip-boundary"
+    boundary="{&quot;top&quot;:0, &quot;bottom&quot;:0, &quot;left&quot;:0, &quot;right&quot;:0}">
```

Comment:
Do the `&quot` get rendered as `"` in the text view? I can play around with this a bit to see if there a better way to write it

Reply:
They don't, they get rendered as is. Based on the demo, this seems to be how we want this to be used so my thought is we would want to show it in the text like this